common/crypto/dkg/src/bte/mod.rs

```diff
@@ -316,7 +245,7 @@ pub fn setup() -> Params {
     let h = hash_g2(b"h", SETUP_DOMAIN);
 
     Params {
-        lambda_t: MAX_EPOCHS_EXP,
+        lambda_t: 1,
```

Comment:
you might have to ask @aniampio here whether we might be able to get rid of `lambda_t` completely